scripts/github_api

```diff
@@ -123,6 +156,24 @@ def open_pull_request(title, src_branch, repo, dst_branch, body=""):
     pr.open(title, body, src_branch, dst_branch)
 
 
+def query_pull_request(repo, number, **kwargs):
+    """
+    Handler for the query_pr command. Returns information about a
+    GitHub pull-request.
+
+    Args:
+        repo (str): target repository path. Format: <repo owner>/<repo>
+        number (int): pull-request number
+        kwargs: keys being queried in the form: key=True
```

Comment:
Why not simply use a list with the queried fields names (as in the SELECT clause of a relational db query)? This would be much easier to the user of this Python method IMHO. 

Maybe you are using this format to make script's command line arguments easier to parse? I'd prefer to not couple GitHub API wrapper and the command line parsing, this will make this code harder to use out of the command line in the future.